store/account/account.go

```diff
@@ -36,6 +36,10 @@ type Store interface {
 	DeleteERC20Storage(address common.Address, from, to int64) error
 	LastSyncERC20Storage(address common.Address, blockNr int64) (result int64, err error)
 
+	// ERC 20 transfer events
+	InsertERC20Transfer(event *model.ERC20Transfer) error
+	DeleteERC20Transfer(address common.Address, from, to int64) error
```

Comment:
how about FindERC20Transfer?

Reply:
We can add it in later PR if we need it.